akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowPrependSpec.scala

```diff
@@ -5,8 +5,6 @@
 package akka.stream.scaladsl
 
 //#prepend
-import akka.stream.scaladsl.Source
```

Comment:
These are used in docs, so it would be good to keep them.

Reply:
Ok